Makefile

```diff
@@ -558,6 +566,41 @@ ifdef LOCAL_BUILD
 	$(error LOCAL_BUILD must not be set for a release)
 endif
 
+release-prep: var-require-all-VERSION-CALICO_VERSION-COMMON_VERSION
+ifdef CNX
+	$(MAKE) var-require-all-CALICO_ENTERPRISE_VERSION
+	yq e ".title = \"$(CALICO_ENTERPRISE_VERSION)\" | .components |= with_entries(select(.key | test(\"^(eck-|coreos-).*\") | not)) |= with(.[]; .version = \"$(CALICO_ENTERPRISE_VERSION)\")" -i config/enterprise_versions.yml
```

Comment:
We should ensure that `yq` is available and is the version that we want; e.g. we use yq2 (yq v2.x) in some projects, and the syntax between them is a bit different. I'd recommend checking that `yq` is available and that the version matches, and, if so, storing that in `$(YQ)` for use elsewhere.

(This has bitten me before in other scripts and Makefiles)

Reply:
updated